database/pgtsv_to_json

```diff
@@ -3,6 +3,29 @@
 # Usage: pgtsv_to_json COLUMN_NAME1:TYPE1 COLUMN_NAME2:TYPE2 ...
 
 import json, sys, csv, re
+from datetime import datetime
+
+def timestamp(timestamp_str):
+    """Given a timestamp string, return a timestamp string in ISO 8601 format to emulate
+    Postgres 9.5's to_json timestamp formatting. The given timestamp is assumed UTC.
+
+    Time zones are not supported. http://bugs.python.org/issue6641
+
+    Examples:
+
+        >>> timestamp('2016-06-17 20:10:38')
+        '2016-06-17T20:10:38+00:00'
+
+        >>> timestamp('2016-06-17 20:10:37.9293')
+        '2016-06-17T20:10:37.929300+00:00'
+
+    """
+
+    try:
+        parsed = datetime.strptime(timestamp_str, '%Y-%m-%d %H:%M:%S.%f')
+    except ValueError:
+        parsed = datetime.strptime(timestamp_str, '%Y-%m-%d %H:%M:%S')
+    return parsed.isoformat() + '+00:00'
```

Comment:
The `pgtsv_to_json` module just takes text over STDIN and isn't aware of any database sessions. Inspecting a separate psql process for state seems brittle, especially since there's no guarantee one exists.

I'm getting the impression that you'd like this module to handle time zone offsets, which is totally reasonable. I like the original module's design, so my preference is to avoid:

- changes to the `pgtsv_to_json` interface,
- changes outside this Python module,
- assumptions about the state of a psql session.

Two options are:

1. Parse and output time zone offsets where they exist on the input, and output no time zone offset where they don't, or
2. Raise a `ValueError` where time zone offsets exist on the input, and output no time zone offsets where they don't.

Either way there's no loss of accuracy, and I'm happy to go in either direction. What do you think?